src/test/TEST_ij/ilu.saved.lassen

```diff
@@ -44,11 +44,11 @@ Final Relative Residual Norm = 6.699477e-09
 
 # Output file: solvers.out.311
 hypre_ILU Iterations = 35
-Final Relative Residual Norm = 8.236857e-09
+Final Relative Residual Norm = 6.392185e-09
```

Comment:
These are pretty significant differences.  Is there a good explanation?

Reply:
Yes, some residuals are quite different. I've looked into this and found out it's due to a few floating-point operations during ILU's setup that were previously carried out on the CPU and are now being done on the GPU. I guess these differences are acceptable in this context.